packages/OmegaPrint-Core.package/OPPrinter.class/instance/FinalStatement.with.with.with..st

```diff
@@ -0,0 +1,9 @@
+actions
+FinalStatement: aNode with: aTerminal with: aStatement with: anotherTerminal
+	
+	| circumflex |
+	circumflex := aTerminal children first.
+	circumflex isNil
+		ifTrue: [^ self value: aStatement]
+		ifFalse: [^ circumflex, ' ', (self value: aStatement)]
```

Comment:
space between circumflex and , missing

Reply:
We added spaces around the circumflexes.

Reply:
I added spaces in the blocks.